lib/ckeditor/config.js

```diff
@@ -9,3 +9,19 @@ CKEDITOR.editorConfig = function (config) {
 	// config.uiColor = '#AADC6E';
 };
 
+CKEDITOR.on('instanceReady', function(event) {

```

Comment:
Please add your changes to the https://github.com/oat-sa/ckeditor-dev and follow instruction to generate release https://github.com/oat-sa/ckeditor-dev#build

Reply:
done